cli.js

```diff
@@ -96,7 +103,7 @@ api.run()
 		logger.exit(api.failCount > 0 || api.rejectionCount > 0 || api.exceptionCount > 0 ? 1 : 0);
 	})
 	.catch(function (err) {
-		if (err instanceof Error) {
+		if (err.name === 'Error') {
```

Comment:
And with that change, we show only `err.message` only in case of `throw new Error()` (e.g. https://github.com/sindresorhus/ava/blob/master/lib/fork.js#L50).